### DISCUSSION TITLE
Severity level of CVE-2023-2253

### DISCUSSION THREAD

**Discussion by oallauddin** [aquasecurity/trivy#4820] [Q&A] [answered]
### Question

The trivy report has CVE-2023-2253 at a severity level of HIGH. 
When I go to the link to review the CVE https://avd.aquasec.com/nvd/cve-2023-2253 it is a MEDIUM.
Why does the trivy report have CSV-2023-2253 at a HIGH severity level?

```
usr/local/bin/crane (gobinary)
==============================
Total: 1 (HIGH: 1, CRITICAL: 0)
┌────────────────────────────────┬───────────────┬──────────┬─────────────────────┬───────────────┬───────────────────────────────────────────┐
│            Library             │ Vulnerability │ Severity │  Installed Version  │ Fixed Version │                   Title                   │
├────────────────────────────────┼───────────────┼──────────┼─────────────────────┼───────────────┼───────────────────────────────────────────┤
│ github.com/docker/distribution │ CVE-2023-2253 │ HIGH     │ v2.8.1+incompatible │ 2.8.2-beta.1  │ DoS from malicious API request            │
│                                │               │          │                     │               │ https://avd.aquasec.com/nvd/cve-2023-2253 │
└────────────────────────────────┴───────────────┴──────────┴─────────────────────┴───────────────┴───────────────────────────────────────────┘
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

Amazon Linux 2023

### Version

```bash
trivy --version
Version: 0.43.1
```

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @oallauddin 
Thanks for your interest to Trivy!

For `Go` we get advisories from `GitHub advisory database` and `vulndb` - https://aquasecurity.github.io/trivy/v0.43/docs/scanner/vulnerability/language/#data-sources.

GitHub has set `High` severity for this CVE. That's why you see `High` severity.
There are cases when OS vendors set different severity for packages they update.

`json` format contains `DataSource` and `SeveritySource` fields to show where Trivy found this vulnerability/severity.

Regards, Dmitriy
